src/functional/geometry/convex_hull.zig

```diff
@@ -204,3 +345,77 @@ test "incremental convex hull random elements" {
         try testing.expectEqual(turn, Turn.left);
     }
 }
+
+test "Compute MABR LinearFunction for known ConvexHull one" {
+    const allocator = std.testing.allocator;
+
+    // Define a set of points forming a simple rectangle.
+    const points = [_]DiscretePoint{
+        .{ .time = 0, .value = 0.0 },
+        .{ .time = 1, .value = 3.0 },
+        .{ .time = 2, .value = 2.0 },
+        .{ .time = 3, .value = 5.0 },
+        .{ .time = 4, .value = 4.0 },
+    };
+
+    // Initialize the convex hull and add points to it.
+    var convex_hull = try ConvexHull.init(allocator);
+    defer convex_hull.deinit();
+
+    for (points) |point| {
+        try convex_hull.addPoint(point);
+    }
+
+    // Calculate the minimum-area bounding rectangle (MABR).
```

Comment:
Seems only the function is returned and not the MABR, maybe update the comment to reflect this.